ambry-store/src/main/java/com/github/ambry/store/BlobStore.java

```diff
@@ -383,6 +393,26 @@ private void checkCapacityAndUpdateReplicaStatusDelegate() {
           }
         }
       }
+      // During startup, we also need to reconcile the replica state from both ZK clusters.
+      if (!started && replicaStatusDelegates.size() > 1 && thresholdBytesLow < index.getLogUsedCapacity()
```

Comment:
nit: this can be `else if` following the statement above.

Reply:
True, but I think maybe it's good to separate it from previous code block. This is a one-off code path which is only called during startup. (In the future, this piece can be removed when migration is complete.)